opencti-platform/opencti-front/src/private/Root.tsx

```diff
@@ -97,6 +97,10 @@ const rootPrivateQuery = graphql`
         }
       }
     }
+    schemaRelationsTypesMapping: schemaRelationsTypesMapping {
```

Comment:
Since it's the same name, could we just have `schemaRelationsTypesMapping {` ?

Reply:
well seen